database_sanitizer/tests/test_dump_mysql.py

```diff
@@ -61,6 +68,30 @@ def test_sanitize_from_stream():
 """ in dump_output_lines
 
 
+def test_skip_table_rows():
+    stream = io.BytesIO(MOCK_MYSQLDUMP_OUTPUT)
+    config = Configuration()
+    config.skip_rows_for_tables.append('test')
+
+    output = list(sanitize_from_stream(stream, config))
+
+    assert "--- Fake MySQL database dump" in output
+    assert """CREATE TABLE `test` (\
+`id` int(11) NOT NULL AUTO_INCREMENT,\
+`created_at` date NOT NULL,\
+`notes` varchar(255) NOT NULL,\
+PRIMARY KEY (`id`)\
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;\
+""" in output
+    assert "--- Final line after `INSERT INTO` statement." in output
+
+    assert """INSERT INTO `test` (`id`, `created_at`, `notes`) VALUES \
+(1,'2018-01-01','Sanitized'),\
+(2,'2018-01-02','Sanitized'),\
+(3,'2018-01-03','Sanitized');\
+""" not in output
```

Comment:
I think you should rather just assert the whole contents of the `output` variable, since the most important part, i.e. this last assert, will break so that it won't test anything anymore, if the insert statement is changed somehow in the input.

Reply:
... or alternatively just make this last assert much simpler and harder to be broken by changes to the input, i.e. something like:
```python
assert "INSERT" not in output
```

Reply:
though that would break to the last line comment which contains word INSERT. (But that comment could be changed of course.) I still think that asserting the whole content would be the cleanest and easiest way.